connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerState.kt

```diff
@@ -15,4 +22,23 @@ internal data class StripeConnectWebViewContainerState(
      * True if the native loading indicator should be visible.
      */
     val isNativeLoadingIndicatorVisible: Boolean = false,
-)
+
+    /**
+     * The appearance to use for the view.
+     */
+    val appearance: Appearance? = null
+) {
+    /**
+     * The background color of the view.
+     */
+    @ColorInt
+    val backgroundColor: Int =
+        appearance?.colors?.background ?: Color.WHITE
+
+    /**
+     * The color of the native loading indicator.
+     */
+    @ColorInt
+    val nativeLoadingIndicatorColor: Int =
+        appearance?.colors?.secondaryText ?: getContrastingColor(backgroundColor, 4.5f)
```

Comment:
nit: since you have the default as `4.5f` and have a comment there, seems like just omitting the param in this fallback case might be better